inference-engine/tests_deprecated/unit/engines/mkldnn/graph/layers/extensions/reduce_tests.cpp

```diff
@@ -4,18 +4,30 @@
 
 #include "test_graph.hpp"
 
```

Comment:
1) As I know contributing to deprecated tests is highly undesirable now. Since we already have [Reduce test](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/shared/src/single_layer_tests/reduce_ops.cpp) in new infrastructure lets extend it with all necessary cases. 
2) As now we support different layouts in Reduce layer we need to cover them in new infrastructure as well. Please use this [MR](https://gitlab-icv.inn.intel.com/inference-engine/dldt/merge_requests/6711/diffs) as an example.

Reply:
Single layer test updated. Thanks Dmitry!

Reply:
@xuchen-intel  I still see changes in deprecated tests. Can we get rid of them?

Reply:
Fixed. Thanks Dmitry!